javascript/javascript2/week1/spirit-animal.js

```diff
@@ -0,0 +1,26 @@
+const userNameInput = document.getElementById('nameInput');
+const inputButton = document.getElementById('inputButton');
+const answerName = document.getElementById('answer');
+
+function getSpiritAnimal() {
+  const spiritAnimal = [
+    'The Bengal Cat',
+    'The Black Bear',
+    'The Calico Cat',
+    'The Canada Jay',
+    'The Capuchin Monkey',
+    'The Clouded Leopard',
+    'The Copperhead Snake',
+    'The Crested Caracara',
+    'The Dromedary Camel',
+    'The German Shepherd',
+  ];
+  if (!userNameInput.value) {
```

Comment:
Nice code and working fine.
But , What if the user types only space in the input box ? 
One solution could be :
if (! userNameInput.value || userNameInput.value.trim().length === 0) . 
or may be you might have better option.